eng/common/Update-Change-Log.ps1

```diff
@@ -12,8 +12,7 @@ param (
   [Parameter(Mandatory = $true)]
   [String]$ChangeLogPath,
   [String]$Unreleased = $True,
```

Comment:
@sima-zhu this PR accidently reverted some other tools changes. We need to be careful when approving the PRs to ensure they only contain the changes we expect to see.

@chidozieononiwu this is an example of why we need the patching so we don't get tools changes entangled with each other.